Description: Fix FTBFS with GCC-15 (Closes: #1096465).
Author: Sven Geuer <sge@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1096465
Forwarded: no
Last-Update: 2025-09-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cowpatty.c
+++ b/cowpatty.c
@@ -61,7 +61,7 @@
 void hexdump(unsigned char *data, int len);
 void usage(char *message);
 void testopts(struct user_opt *opt);
-void cleanup();
+void cleanup(int unused);
 void parseopts(struct user_opt *opt, int argc, char **argv);
 void closepcap(struct capture_data *capdata);
 void handle_dot1x(struct crack_data *cdata, struct capture_data *capdata,
@@ -91,7 +91,7 @@
 	       "\t-V \tPrint program version and exit\n" "\n");
 }
 
-void cleanup()
+void cleanup(int unused)
 {
 	/* lame-o-meter++ */
 	sig = 1;
--- a/genpmk.c
+++ b/genpmk.c
@@ -53,7 +53,7 @@
 		"argument.\n");
 }
 
-void cleanup()
+void cleanup(int unused)
 {
 	/* lame-o-meter++ */
 	sig = 1;
